Formula/rxvt-unicode.rb

```diff
@@ -31,6 +32,11 @@ class RxvtUnicode < Formula
 
   uses_from_macos "perl"
 
+  resource "libptytty" do
```

Comment:
How do we know which commit hash to use?

Reply:
That's a great question - the answer is to use the official upstream tarball which is tagged rather than this GitHub archive: http://dist.schmorp.de/libptytty/libptytty-2.0.tar.gz.  I copied this from another PR so I don't know why it didn't use that official tarball but I am going to change it so that it does now.

Reply:
Switching to the official tarball worked with no issues on ARM macOS and Linux.  I think this should be ready when the Intel macOS runners are done.

Reply:
It would probably still be better to turn this into a separate formula.